Sources/TelemetryClient/Signal.swift

```diff
@@ -38,26 +38,29 @@ internal struct SignalPostBody: Codable, Equatable {
     let isTestMode: String
 }
 
-internal struct DefaultSignalPayload: Encodable {
-    let platform = Self.platform
-    let systemVersion = Self.systemVersion
-    let majorSystemVersion = Self.majorSystemVersion
-    let majorMinorSystemVersion = Self.majorMinorSystemVersion
-    let appVersion = Self.appVersion
-    let buildNumber = Self.buildNumber
-    let isSimulator = "\(Self.isSimulator)"
-    let isDebug = "\(Self.isDebug)"
-    let isTestFlight = "\(Self.isTestFlight)"
-    let isAppStore = "\(Self.isAppStore)"
-    let modelName = Self.modelName
-    let architecture = Self.architecture
-    let operatingSystem = Self.operatingSystem
-    let targetEnvironment = Self.targetEnvironment
-    let locale = Self.locale
-    var extensionIdentifier: String? = Self.extensionIdentifier
```

Comment:
This one property was a `var` before, but I found no existing need to mutate it, the tests pass, and the property was previously internal, so I figured it was safe to change to `let`.